docs/preview/features/secrets/secret-store-integration.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
More input can be pulled from the blog post when it's ready but this is good for now.